tests/s25Main/lua/testLuaConstants.cpp

```diff
@@ -0,0 +1,229 @@
+// Copyright (c) 2021 - 2021 Settlers Freaks (sf-team at siedler25.org)
+//
+// This file is part of Return To The Roots.
+//
+// Return To The Roots is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 2 of the License, or
+// (at your option) any later version.
+//
+// Return To The Roots is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Return To The Roots. If not, see <http://www.gnu.org/licenses/>.
+
+#include "GameWithLuaAccess.h"
+#include <boost/test/unit_test.hpp>
+
+struct LuaConstsTestsFixture : public LuaTestsFixture
+{
+    LuaConstsTestsFixture()
+    {
+        initWorld();
+        // This adds an `elements` function which can iterate over a table generated by
+        // table.pack (important as nil entries caused by missing constants abort regular iteration)
+        executeLua(R"LUA(
```

Comment:
gcc-6 does not have support for raw strings yet?

Reply:
or at least broken support in macros, could that be? see https://gcc.gnu.org/bugzilla/show_bug.cgi?id=55971

Reply:
Strange... Can't reproduce on godbolt: https://godbolt.org/z/ddKxKo
Doing a quick check, the we can decide if we simply remove support for GCC 6 (which I can't even install anymore on Ubuntu 20) for lack of proper C++11 support or just remove the macro (not that great, but a comment will suffice)

Reply:
perhaps its 6.0 on the pipeline?

Reply:
Cmake says 6.5 (which is odd as Godbolt has only 6.4, oh and GCC x.0 doesn't exist. They start at x.1 and 6.1 is also working on godbolt). Anyway, removed the macro call.